py3status/modules/mega.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+"""
+Display status of MEGAcmd.
+
+Configuration parameters:
+    cache_timeout: refresh interval for this module (default 10)
+    format: display format for this module (default "MEGA: {format_sync}")
+    format_sync: display format for every sync (default "{SyncState}")
+    format_sync_separator: show separator if more than one sync (default " ")
+
+Format placeholders:
+    {format_sync} Format for every sync returned by 'mega-sync' command.
+
+format_sync placeholders:
+    Any column returned by 'mega-sync' command - in lower case!
+    For example: id, syncstate, localpath
```

Comment:
I purposefully don't want to list placeholders because this module doesn't rely on them or control them, people shouldn't rely on this description because it will quickly be outdated. If a new column is added in upstream, or an existing column is updated, our own documentation will get out of sync and people will be confused and will be filing issues. 

It is not that hard to execute `mega-sync` in terminal and get up-to-date list of columns, and it is documented so it's not like people don't know where to get the available placeholders 😉 

If other modules list placeholders that they don't really rely on and control, they should be fixed too and explain to users how to get up to date info.

Reply:
It's same with `nvidia_smi` for me, but I make a list of (some) placeholders users might like...

>It is not that hard to execute mega-sync in terminal and get up-to-date list of columns

Also, I had to scroll up and down this issue to find a placeholder because I was looking for that one placeholder you mentioned somewhere... It was `{size}`.

It's nice for users not using `mega` module to be able to find a list of placeholders without installing and running `mega` command. We might have to make an account too. *cough*

Also, I want to fix this up so it'll be ready to go in. It's fine if we miss some. I didn't include other 20+ placeholders from `nvidia_smi` as it depends on what gpu users have.

>because it will quickly be outdated.

Is it still up to date? Anything changed? Headers?

Reply:
I don't see why users would be interested in available placeholders of a mega module if they don't even have a mega account. You as a reviewer of the initial PR are an exception 😉 

megacmd got a new major version since the time I submitted the PR, I didn't track if the headers change or not, but as a maintainer of this module I'm happy that I don't even need to worry about it.